ecomerce-app/src/backend/db/products.js

```diff
@@ -5,26 +5,51 @@ import { v4 as uuid } from "uuid";
  * You can add products of your wish with different attributes
  * */
 
+
+//  image:
+//  "https://res.cloudinary.com/diyo6qdy1/image/upload/v1647612850/e-comm/man_bgremove/5-removebg-preview_seobft.png",
+// title: "FERARI Black",
+// price: "1800",
+// mrpPrice: "3800",
+// rating: 4,
+// brand: "Astoria",
+// inStock: true,
+// fastDelivery: true,
+// offer: ["60% off"],
+// categoryName: "man",
+// color: "brown",
+// stock: 22,
+// isLiked: false,
+// outOfStock: false,
+// isSelected: false,
+
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    img: "https://res.cloudinary.com/e-comerce/image/upload/v1648542477/samples/asspee_arng45.png",
+    title: "tools",
+    mrpPrice: "5000",
+    discountPrice:4500,
+    categoryName: "Spray Machine",
+    discount:["10% off"]
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    img: "https://res.cloudinary.com/e-comerce/image/upload/v1648542517/samples/ecomerce/assest/seedcard_vke0ta.jpg",  
+    title: "tools",
+    mrpPrice: "3500",
+    discountPrice:3150,
+
+    categoryName: "Spray Machine",
+    discount:["10% off"]
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    img: "https://res.cloudinary.com/e-comerce/image/upload/v1648542516/samples/ecomerce/assest/ferilizercard_jhklrc.jpg",
```

Comment:
you can assign this (https://res.cloudinary.com/e-comerce/image/upload/v1648542516/samples/ecomerce/) part to a const variable so that no matter how many products you have you do not have to write the whole URL again and again.

Reply:
thank you i will do